content/docs/guides/go-application.md

```diff
@@ -10,7 +10,21 @@ NOTE: For comprehensive API documentation, see the [GoDoc](https://godoc.org/git
 
 ## Installation
 
-You can install the `prometheus`, `promauto`, and `promhttp` libraries necessary for the guide using [`go get`](https://golang.org/doc/articles/go_command.html):
+To integrate Prometheus with your Go application, you'll need the `prometheus`, `promauto`, and `promhttp` libraries. Starting with Go 1.17, Go Modules are required for managing dependencies.
```

Comment:
Thanks for this, but I'm afraid the more we add, the more maintenance it will require.
Let's just replace the existing link https://golang.org/doc/articles/go_command.html with with https://go.dev/ref/mod#go-get which is more module oriented.

And to be honest the warning when using `go get` outside a module is clear enough:

```
go: go.mod file not found in current directory or any parent directory.
	'go get' is no longer supported outside a module.
	To build and install a command, use 'go install' with a version,
	like 'go install example.com/cmd@latest'
	For more information, see https://golang.org/doc/go-get-install-deprecation
	or run 'go help get' or 'go help install'.
```